src/seedu/addressbook/commands/DeleteCommand.java

```diff
@@ -19,11 +20,13 @@
 
     public static final String MESSAGE_DELETE_PERSON_SUCCESS = "Deleted Person: %1$s";
 
-
-    public DeleteCommand(int targetVisibleIndex) {
+    public DeleteCommand(int targetVisibleIndex, UndoDeleteCommand undoDelete) {
         super(targetVisibleIndex);
     }
 
+    public DeleteCommand(int targetVisibleIndex) { super(targetVisibleIndex);
```

Comment:
Coding standard violation!
Please check how to write methods with a single line of code.